### DISCUSSION TITLE
Convert multipage PDF to separate TIFF files

### DISCUSSION THREAD

**Discussion by winvis** [ImageMagick/ImageMagick#4996] [Help]
Hi,

I know a parameter like %01d works to convert a multipage pdf to seperate tiff files.
But the first page gets a filename  like ocr_0.tiff for the first page, ocr_1.tiff for the second and so on.

This is no problem for a computer, but I need pagenumbers that humans can understand too. So the first page should be ocr_1.tiff.

Renaming the files after conversion is not the best option... It looks simple, but so far I haven't found it. 
How should I do that? 

Regards, Gerrit

**Comment by fmw42** [comment]
Add `-scene 1` to your command

**Comment by winvis** [comment]
Hi,

This works fine now! Thanks for sharing!

Gerrit
